test/controllers/declaration/RepresentativeDetailsPageControllerSpec.scala

```diff
@@ -256,17 +256,39 @@ class RepresentativeDetailsPageControllerSpec extends CustomExportsBaseSpec with
       header.headers.get("Location") must be(Some("/customs-declare-exports/declaration/additional-actors"))
     }
 
-    "accept form with status and address only" in {
+    "accept form with status and EORI if on standard journey" in {
       withCaching[RepresentativeDetails](None)
       withCaching[Choice](Some(Choice(Choice.AllowedChoiceValues.SupplementaryDec)), choiceId)
 
-      val result = route(app, postRequest(uri, correctRepresentativeDetailsAddressOnlyJSON)).get
+      val result = route(app, postRequest(uri, correctRepresentativeDetailsEORIOnlyJSON)).get
       val header = result.futureValue.header
 
       status(result) must be(SEE_OTHER)
       header.headers.get("Location") must be(Some("/customs-declare-exports/declaration/additional-actors"))
     }
 
+    "accept form with status and address only if on supplementary journey" in {
+      withCaching[RepresentativeDetails](None)
+      withCaching[Choice](Some(Choice(Choice.AllowedChoiceValues.StandardDec)), choiceId)
+
+      val result = route(app, postRequest(uri, correctRepresentativeDetailsAddressOnlyJSON)).get
+      val header = result.futureValue.header
+
+      status(result) must be(SEE_OTHER)
+      header.headers.get("Location") must be(Some("/customs-declare-exports/declaration/consignee-details"))
```

Comment:
Here, as far as I remember, the next page should be different.

Reply:
yes this was a copy/paste issue on the user story. I have spoken with the BAs and I have amended that.